packages/odyssey-react-mui/package.json

```diff
@@ -51,6 +49,9 @@
     "typescript": "^4.4.4"
   },
   "peerDependencies": {
+    "@emotion/react": "^11",
+    "@mui/icons-material": "^5",
+    "@mui/material": "^5",
     "react": ">=16 <18",
```

Comment:
I thought you mentioned mui5 needs react 17. 
https://mui.com/material-ui/getting-started/installation/